crates/biome_configuration/src/analyzer/linter/mod.rs

```diff
@@ -51,6 +52,38 @@ pub enum RuleDomainValue {
     Recommended,
 }
 
+#[derive(Clone, Debug, Default, Deserialize, Deserializable, Eq, PartialEq, Serialize, Merge)]
+pub struct RuleDomains(FxHashMap<RuleDomain, RuleDomainValue>);
```

Comment:
Can you add `Ord` and `PartialOrd` to the domains? It will help fix an issue on the website

Reply:
Added in [a892a28](https://github.com/biomejs/biome/pull/5306/commits/a892a289beff0927125da990a39b65e3172faddc), is it correct?

Reply:
It seems so: https://github.com/biomejs/website/pull/1797/files#diff-cb306886ea92d37b5ce007cc39b3fcf26079234cd572cc7502e05696b2d75346R67-R69